src/wmtk/Mesh.hpp

```diff
@@ -158,6 +185,10 @@ class Mesh
     }
     Tuple switch_edge(const Tuple& tuple) const { return switch_tuple(tuple, PrimitiveType::Edge); }
     Tuple switch_face(const Tuple& tuple) const { return switch_tuple(tuple, PrimitiveType::Face); }
+    Tuple switch_tetrahedron(const Tuple& tuple) const
```

Comment:
i'd like to remove all implementations of functions from the class definitions - my preference is that if you're going to force inlining then just put it at the end of the header (or in a `Mesh_impl.hpp` that is included at the bottom of the header)

Reply:
this is a bit more visible in this case because your formatter decided to make this function take 3 lines instead of the previous singel line ones - but i'd like to remove those as well TBH

Reply:
I moved the implementations to the end of the header 👍